scripts/ErrorReporter/errorreport.py

```diff
@@ -108,8 +108,8 @@ def get_plain_text_edit_field(self, text_edit, expected_type):
 
     def check_placeholder_text(self):
         if not self.free_text_edited:
-            self.input_free_text.setPlainText("")
             self.free_text_edited = True
+            self.input_free_text.setPlainText("")
```

Comment:
This looks like it is because Qt4 doesn't have the `setPlaceHolderText` option so mimics it with this function and calling set again calls this unless `free_text_edited` is True.

Reply:
This was a thing I implemented as one of my first issues. What I don't understand is why it suddenly stopped working. I guess something changed in the move to python 3?

Reply:
I guess it must have. It's odd as looking at the code it seems like it should have been broken before but we'll just have to chalk it up to one of those things.